jenkins/broadcom/buildimage-brcm-201904/Jenkinsfile

```diff
@@ -35,8 +35,11 @@ pipeline {
 git submodule foreach --recursive '[ -f .git ] && echo "gitdir: $(realpath --relative-to=. $(cut -d" " -f2 .git))" > .git'
 
 make configure PLATFORM=broadcom
+
+SONIC_OVERRIDE_BUILD_VARS="SIGNING_KEY=/tmp/certs/signing.key SIGNING_CERT=/tmp/certs/signing.cert CA_CERT=/tmp/certs/ca.cert"
+DOCKER_BUILDER_MOUNT="$(pwd):/sonic -v /nfs/build/test-certs:/tmp/certs"
```

Comment:
>-v [](start = 36, length = 2)

Suggest put certs under files/image_config/secureboot/test-certs, so you don't need to mount extra path.

Reply:
Does someone have chance to access the workspace to access the certificates?
If the local branch is not clean, it will have impact on image version, right? Do we need to add a gitignore for the folder?

Reply:
I see your point to place in /tmp. That will be good.

---
In reply to: [462594438](https://github.com/Azure/sonic-build-tools/pull/140#discussion_r462594438) [](ancestors = 462594438)